src/ui.py

```diff
@@ -172,11 +172,17 @@ def show_about_window(self):
     Adw.AboutWindow(
         transient_for=self.get_window(), application_name=self.get_name(),
         application_icon=self.get_application_id(), website=self.get_website(),
-        developer_name=self.get_author(), issue_url=self.get_issues(),
+        developer_name=self.get_author() + ", et al.",
+        issue_url=self.get_issues(),
         version=self.get_version(), developers=[
             "Sjoerd Stendahl <contact@sjoerd.se>",
             "Christoph Kohnen <christoph.kohnen@disroot.org>",
         ],
+        designers=[
+            "Sjoerd Stendahl <contact@sjoerd.se>",
+            "Christoph Kohnen <christoph.kohnen@disroot.org>",
+            "Tobias Bernard",
```

Comment:
Perhaps add email here as well for consistency's sake. (From [here](https://tobiasbernard.com/))

```suggestion
            "Tobias Bernard", <hi@tobiasbernard.com>",
```

Reply:
No objection to being mentioned, though I'd prefer to have my GNOME email address in there (tbernard@gnome.org) :)